scripts/python/packages/parthenon_tools/parthenon_tools/movie2d.py

```diff
@@ -36,14 +65,30 @@ def read(filename, nGhost=0):
 
 
 def plot_dump(
-    xf, yf, q, name, with_mesh=False, block_ids=[], xi=None, yi=None, xe=None, ye=None
+    xf,
+    yf,
+    q,
+    name,
+    with_mesh=False,
+    block_ids=[],
+    xi=None,
+    yi=None,
+    xe=None,
+    ye=None,
+    components=[0, 0],
 ):
 
     if xe is None:
         xe = xf
     if ye is None:
         ye = yf
 
+    # get tensor components
```

Comment:
Could we add an `assert len(q.shape) <= 5` here, since technically rank-3 tensor data could arrive from the dump file?

Reply:
Good idea :+1: